### DISCUSSION TITLE
Curse of tungues

### DISCUSSION THREAD

**Discussion by stefment** [Zidras/DBM-Warmane#202] [Ideas]
Hello, curse of tungues ruin some timers on some bosses, for example sindragosa. After a while you probably learn to adjust, but ive had some timing issues with getting stacks to expire before the boss pulls me in, because the countdown is off :) Its not a big deal but i was wondering if a curse of tungue bonus timer could be possible if the addon detected curse of tungues on the boss?

**Comment by Zidras** [comment]
Even with curse detection, timers have variance. Unless you are able to give a condition that corrects timers correctly 100% of the time, I have to consider this wishful thinking
